server/routes/activities/allocation-dashboard/handlers/allocationDashboard.test.ts

```diff
@@ -280,9 +348,18 @@ describe('Route Handlers - Allocation dashboard', () => {
     })
 
     it('should return correct candidates with risk level filter set to any', async () => {
+      const tomorrow = addDays(today, 1)
```

Comment:
It's test code so doesn't really matter, but since this value is used by several tests it might be useful to define this up with the `today` const so it can be reused.